packages/paste-core/components/tooltip/stories/index.stories.tsx

```diff
@@ -1,108 +1,115 @@
-import {withKnobs} from '@storybook/addon-knobs';
+import {withKnobs, text} from '@storybook/addon-knobs';
 import {Absolute} from '@twilio-paste/absolute';
 import {Box} from '@twilio-paste/box';
 import {Button} from '@twilio-paste/button';
 import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {Stack} from '@twilio-paste/stack';
 import {Text} from '@twilio-paste/text';
 import {Tooltip} from '../src';
 
-const Example: React.FC<{}> = () => {
-  return (
-    <Tooltip text="Welcome to Paste!">
-      <Button variant="primary">Open Tooltip</Button>
-    </Tooltip>
-  );
-};
-
-const IconButtonExample: React.FC<{}> = () => {
-  return (
-    <Tooltip text="Welcome to Paste!">
-      <Button variant="secondary" size="icon">
-        <InformationIcon decorative={false} title="Open Tooltip" />
-      </Button>
-    </Tooltip>
-  );
-};
-
-const CustomExample: React.FC<{}> = () => {
-  return (
-    <Box display="flex" alignItems="center">
-      <Text as="span">Tooltip should appear from the icon.</Text>
-      <Tooltip text="Welcome to Paste!">
-        <a href="https://paste.twilio.design" style={{display: 'inline-flex'}}>
-          <InformationIcon decorative={false} title="Open Tooltip" />
-        </a>
-      </Tooltip>
-    </Box>
-  );
-};
-
-const BottomExample: React.FC<{}> = () => {
-  return (
-    <Tooltip text="Welcome to Paste!" placement="bottom">
-      <Button variant="primary">Open Tooltip</Button>
-    </Tooltip>
-  );
-};
-
-const TopExample: React.FC<{}> = () => {
-  return (
-    <Absolute preset="bottom" bottom={12}>
-      <Tooltip text="Welcome to Paste!" placement="top-start">
-        <Button variant="primary">Open Tooltip</Button>
-      </Tooltip>
-    </Absolute>
-  );
-};
-
-const RightExample: React.FC<{}> = () => {
-  return (
-    <Tooltip text="Welcome to Paste!" placement="right-start">
-      <Button variant="primary">Open Tooltip</Button>
-    </Tooltip>
-  );
-};
-
-const LeftExample: React.FC<{}> = () => {
-  return (
-    <Absolute preset="right" top={12} right={12}>
-      <Tooltip text="Welcome to Paste!" placement="left-start">
-        <Button variant="primary">Open Tooltip</Button>
-      </Tooltip>
-    </Absolute>
-  );
-};
-
 storiesOf('Components|Tooltip', module)
   .addDecorator(withKnobs)
   .add('Default', () => {
-    return <Example />;
-  })
-  .add('Tooltip Top', () => {
-    return <TopExample />;
+    return (
+      <Tooltip text={text('text', 'Welcome to Paste!')}>
+        <Button variant="primary">Open Tooltip</Button>
+      </Tooltip>
+    );
   })
-  .add('Tooltip Left', () => {
-    return <LeftExample />;
+  .add('Tooltip Placements', () => {
+    return (
+      <Box marginTop="space120" paddingTop="space120" paddingLeft="space60">
+        <Stack orientation="horizontal" spacing="space40">
+          <Tooltip text="Welcome to Paste!" placement="top">
+            <Button variant="primary">Open Tooltip</Button>
```

Comment:
Ultra nit but can you give these button texts the placement too? Like `Open to the top/right/left`

Reply:
Done!